### PR TITLE
ci: Run iwyu on all src files

### PR DESCRIPTION
**Pull request by maflcko**
This makes it easier to look at the CI output of a file without having to manually add it first to the list.

### CODE REVIEW

**Comment by fanquake** [comment]
Concept ACK on removing the continual manual updating of this list

**Comment by hebasto** [comment]
> > (qt/qrc_bitcoin_locale.cpp has correct #includes/fwd-decls)
> 
> Looks like it's mostly working, except GUI stuff is still being dragged in

And `qt/qrc_bitcoin.cpp` as well.

File: ci/test/06_script_b.sh [hunk 1]
```diff
@@ -42,51 +42,16 @@ if [ "${RUN_TIDY}" = "true" ]; then
   set -eo pipefail
   cd "${BASE_BUILD_DIR}/bitcoin-$HOST/src/"
   ( run-clang-tidy-16 -quiet "${MAKEJOBS}" ) | grep -C5 "error"
+  # Filter out files by regex here, because regex may not be
+  # accepted in src/.bear-tidy-config
+  # Filter out:
+  # * qt qrc and moc generated files
+  # * walletutil (temporarily)
```
Comment (hebasto) [inline ci/test/06_script_b.sh:49]: Mind elaborating?

**Review by hebasto — APPROVED** [review]
ACK ddddf4957b02c83ed9b6c46b35d8ae1e137889d2

**Pull request merged**